.github/workflows/release.yml

```diff
@@ -57,4 +60,4 @@ jobs:
       run: cosign sign "ghcr.io/linkerd/proxy-init:$TAG"
     - name: Prune docker layers cache
       shell: bash
-      run: bin/docker-cache-prune
+      run: just docker-cache-prune
```

Comment:
Does this mean that the script is still hanging around? Do we want both? Should just call the script? Or should we delete the script?

Reply:
When re-doing the CI I wanted to introduce the scripts but instead went with just. The scripts were never committed but the release flow was, and I forgot to change the ref to the inexistent script here.